includes/wikia/services/WAMService.class.php

```diff
@@ -89,6 +89,42 @@ public function getCurrentWamScoreForWiki ($wikiId) {
 	}
 
 	/**
+	 * Returns the latest WAM rank provided a wiki ID
+	 * @param int $wikiId
+	 * @return number
+	 */
+	public function getCurrentWamRankForWiki ($wikiId) {
```

Comment:
This code should be spacey per our [PHP coding conventions](https://www.mediawiki.org/wiki/CC/PHP).